projects/mysadna/package.json

```diff
@@ -22,6 +22,7 @@
     "@angular/router": "~7.1.0",
     "angularfire2": "^5.1.1",
     "core-js": "^2.5.4",
+    "firebase-tools": "^6.3.1",
```

Comment:
Why did you install firebase-tools?

Reply:
I think it's excess

Reply:
I had to, in order to use Firebase hosting, of course.

Reply:
Or should I delete that too? and if I do, should I also delete it from `package-lock.json`?

Reply:
Yes, delete it please. Probably you installed it by accident. Because to use firebase you need to install firebase-tools globally, not into project.
```
npm install -g firebase-tools
```
-g means global

Reply:
You don't need to additionally delete from package-lock.json

Reply:
Just run:
```
npm rm firebase-tools
```